System.IO.IOException: The process cannot access the file 'D:\TDA_intern\Projects\DatabaseApp\Product_Mangement_App-master\Logs\Receipts\ Order C00007_test2 status.XLSX' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.File.InternalDelete(String path, Boolean checkHost)
   at System.IO.File.Delete(String path)
   at DatabaseApp.OrderManagementDetail.btnPrint3_Click(Object sender, RoutedEventArgs e) in D:\TDA_intern\Projects\DatabaseApp\Product_Mangement_App-master\DatabaseApp\OrderManagementDetail.xaml.cs:line 982
